tools/gn

```diff
@@ -218,6 +219,9 @@ def to_gn_args(args):
     elif args.goma and os.path.exists(goma_home_dir):
       gn_args['use_goma'] = True
       gn_args['goma_dir'] = goma_home_dir
+    elif args.goma and os.path.exists(depot_tools_bin_dir):
```

Comment:
This would have to go above the `args.goma` case wouldn't it?

Reply:
I think this is correct. Theres 3 cases:
- Priority 1: args.goma and goma_dir is present.
- Priority 2: args.goma and goma_home_dir is present.
- Priority 3: args.goma and (none of the other two are present + depot_tools dir is present.)